test/regressions/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@mui-x-internal/test-regressions",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "cross-env NODE_ENV=production pnpm build && concurrently --success first --kill-others \"pnpm run mocha\" \"pnpm run server\"",
+    "build": "cross-env E2E_BUILD=true vite build",
+    "dev": "concurrently \"pnpm run build --watch\" \"pnpm run server\"",
```

Comment:
It looks like `pnpm release:build` needs to be run for this to work, which makes it a DX regression.
Can we keep using aliases for regression and e2e tests in dev mode?

Reply:
For the dev script to have proper HMR it would need to use `vite dev` rather than `vite build`, but that command triggers yet another build issue with very unclear causes. I'd leave as is and keep that as a follow-up.